date-input.html

```diff
@@ -14,6 +14,7 @@
 <link rel="import" href="../paper-input/paper-input-container.html">
 <link rel="import" href="../paper-styles/default-theme.html">
 <link rel="import" href="../paper-styles/typography.html">
+<link rel="import" href="../iron-a11y-keys/iron-a11y-keys.html">
```

Comment:
You should use the `iron-a11y-keys-behavior` instead; it's much more performant.